databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/factory/DatabusFactory.java

```diff
@@ -47,6 +66,82 @@ public DatabusClient create() {
                 _log.error("Databus discovery startup failed", e);
             }
         }
-        return new DatabusClient(_emoServiceDiscovery, _emoClient, _apiKey);
+        return new DatabusClient(_emoServiceDiscovery, _emoClient, _apiKey, createRetryPolicy());
+    }
+
+    public RetryPolicy<Object> createRetryPolicy() {
```

Comment:
this method is duplicated in all jersey2 clients.
any objections to moving to the emodb-common-jersey2-client maven module?